source/loader/layers/sanitizer/ur_sanitizer_utils.cpp

```diff
@@ -72,6 +72,20 @@ ur_device_handle_t GetDevice(ur_queue_handle_t Queue) {
     return Device;
 }
 
+std::string GetDeviceName(ur_device_handle_t Device) {
+    size_t Size = 0;
+    [[maybe_unused]] auto Res = context.urDdiTable.Device.pfnGetInfo(
+        Device, UR_DEVICE_INFO_NAME, 0, nullptr, &Size);
+    assert(Res == UR_RESULT_SUCCESS);
```

Comment:
NIT: Adding more information with the assert like above implementation?

Reply:
Done